### DISCUSSION TITLE
Edit Override

### DISCUSSION THREAD

**Discussion by PieTheSurgeon** [silveredgold/beta-protection#43] [Ideas]
When using beta protect on your main computer it isn't always practical to have it in the "enabled" mode but having an override is so much fun. So the best choice is "on demand" so specific sites can be blocked. The issue is when making a new override it takes forever to add all the previous "forced" sites and setting. I would love a feature to upload a previously made override and be able to edit it then export as a new save. This would not affect an override you are currently in, just one saved for future use.

**Comment by silveredgold** [comment]
I've just added this feature and it should be available in 0.2.5+
